My Android autotest

chibert opened the pull request.

chibert committed:
My solution (version 01)

My solution of the task (version 01).

chibert committed:
Merge remote-tracking branch 'refs/remotes/origin/Chibert'

chibert committed:
My solution (version 02)

My solution (version 02):
- Убрал комментарии в class MyFirstTest.
- Убрал лишнюю зависимость в commons-module pom.xml.
- Убрал из проекта junit-4.12.jar.
- Отформатировал код в class UserSteps.

chibert committed:
My solution (version 03)

My solution (version 03):
- Реализация тестов в class MyFirstTest при помощи параметризации.
- Соответсвующие изменения метода shouldBeAtPlace в class UserSteps
(теперь передаем методу Place и получаем UserSteps).

chibert committed:
My

My

chibert committed:
Repo tuning

steps-module
+ ru.qatools.school.rules.MobileDriverRule
+ ru.qatools.school.screens.MainScreen.java
+ ru.qatools.school.screens.SelectStationScreen.java
+ ru.qatools.school.steps.mobilesteps

commons-module
+ ru.qatools.school.mobiletests

chibert committed:
Updates + 3 Steps + 1 Test

steps-module
Update ru.qatools.school.screens.MainScreen.
Update ru.qatools.school.screens.SelectStationScreen.
Update ru.qatools.school.rules.MobileDriverRule.
Update + 3 Steps in ru.qatools.school.steps.mobilestepsDefaultSteps:
+ clickOn
+ enterText
+ shouldSeeTimeLongerThan

commons-module
Update + 1 Test in ru.qatools.school.mobiletests.AndroidMetroTest:
+ shouldSeeCorrectTimeTravel

totallynotkate commented on steps-module/src/main/java/ru/qatools/school/screens/MainScreen.java:
```diff
@@ -13,23 +13,32 @@
     }
 
-    @Name("������� ����� ����������")
+    @Name("���� ����� ��������� ������� - From")
```
Научи и меня клингонскому =)

totallynotkate commented on steps-module/src/main/java/ru/qatools/school/steps/mobilesteps/DefaultSteps.java:
```diff
@@ -1,43 +1,46 @@
+        Pattern pattern = Pattern.compile("\\d+");
+        Matcher matcher = pattern.matcher(timeText);
+        matcher.find();
+        int timeMinut = Integer.parseInt(matcher.group());
```
Не сработает со временем поездки 60+ минут

chibert committed:
DrWeb

Dr Web first test.

chibert committed:
upgrade test + 1 new test

+ AntispamScreen
+ AntitheftScreen
+ UrlFilterScreen
+ test with them

chibert committed:
+Screens & Tests

+ SecurityAuditorScreen
+ FirewallScreen

+ AntitheftScreen tests
+ some Firewall and SecurityAuditor
